### PR TITLE
BXMSDOC-5028 Standardize OptaPlanner name

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

**Comment by emmurphy1** [comment]
@ge0ffrey ready for your review.

File: optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc [hunk 1]
```diff
@@ -8,22 +8,19 @@
...
 <<explainingTheScore,Constraint justifications>> may not function properly.
+//COMMENT: Lukas, what specifically  might not function.  This link is to section 5.6, Explaining the score: which constraints are broken?  Or maybe we don't need to worry about this based on your comment below?
 ====
 
 // TODO Remove the previous sentence when PLANNER-1709 is fixed.
 
-To use constraint streams in your project, implement the `ConstraintProvider` interface and use the following in your
-solver config:
+To use constraint streams in your project, implement the `ConstraintProvider` interface and inlcude the following code in your solver configuration:
+//COMMENET: Is adding the code how you implement `ConstraintProvider` or do I need to do both: Implement `ConstraintProvider` and add the code to my solve configuration?
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc]: "COMMENET": if it means comment, then the "//" already imply that.
Answer: You need both: Implement a class that implements ConstraintProvider and add it in the solver configuration.

Can we move this change into a separate PR because this is a cross-cutting all-files PR?
(I don't mind if small PR's do multiple changes, but cross-cutting all-files PR I prefer isolated to avoid missing a change).

File: optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc [hunk 2]
```diff
@@ -88,29 +85,30 @@ image::ConstraintStreams/constraintStreamJustification.png[align="center"]
...
 [[constraintStreamsFilter]]
-== Filter
+== Constraint streams API filter
 
 image::ConstraintStreams/constraintStreamFilter.png[align="center"]
 
 
 [[constraintStreamsJoin]]
-== Join
+== Constraint streams join BLAH
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc]: BLAH?

File: optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ConferenceScheduling/conferenceSchedulingArchitecture.svg [hunk 3]
```diff
@@ -1705,7 +1705,7 @@
            y="150.00014"
            style="font-size:14.66666698px;text-align:center;text-anchor:middle" /></flowRegion><flowPara
          id="flowPara4308-6"
-         style="font-size:14.66666698px;line-height:1.25">Planner works with plain Java objects that are read/written to *.xlsx</flowPara></flowRoot>    <image
+         style="font-size:14.66666698px;line-height:1.25">OptaPlanner works with plain Java objects that are read/written to *.xlsx</flowPara></flowRoot>    <image
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ConferenceScheduling/conferenceSchedulingArchitecture.svg]: When SVG's are changed, their PNG should be re-exported. Furthermore this increases the size of the text box, so it might affect the layout of the diagram (let's hope it doesn't).
I suggest we meet up for the Inkscape tutorial first and then handle SVG's in a separate PR that also changes the PNG's (not just here, but also in optaplanner-website)?

Comment (emmurphy1) [reply optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ConferenceScheduling/conferenceSchedulingArchitecture.svg]: Yes, I didn't consider the text in images. As we move forward, let's consider moving as much text as possible (titles, descriptions) out of the images an into the asciidoc text.

File: optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc (hunk 1 above)
Comment (emmurphy1) [reply optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc]: My bad. I have reinstated the original constraint streams file and check for Planner.

**Comment by emmurphy1** [comment]
@ge0ffrey would you please glance through the files again to make sure my changes are now correct?

File: optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ConferenceScheduling/conferenceSchedulingArchitecture.svg (hunk 3 above)
Comment (ge0ffrey) [reply optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ConferenceScheduling/conferenceSchedulingArchitecture.svg]: Can we remove the SVG change from the PR if it doesn't include the PNG change?
If the SVG file hasn't been opened, there is no guarantee the layout isn't corrupted

**Comment by ge0ffrey** [comment]
Let's either do the SVG and PNG change or neither (= postpone to other PR). I hope that makes sense.

File: optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ConferenceScheduling/conferenceSchedulingArchitecture.svg (hunk 3 above)
Comment (ge0ffrey) [reply optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ConferenceScheduling/conferenceSchedulingArchitecture.svg]: Adjusted after merge with this change:
https://github.com/kiegroup/optaplanner/commit/277edf1ba7d0d62b96100071c1e1f33c94c77f64

**Comment by ge0ffrey** [comment]
Merged, thanks @emmurphy1 :)
